### ISSUE TITLE
Could you please provide in single Exe.

### ISSUE THREAD

**Issue by mskumaran** [jimradford/superputty#522]
everyone wants to be used in Single file as SuperPutty.exe instead of Zip file/Folder with code and additional file. Could you please make it visible in download option additionally.
Thanks.....

**Comment by ericfelker** [comment]
I have the same request. I am unfamiliar with Git, and have been struggling for a good while to understand how I take the downloaded zip and get superputty installed from it. Do I need Git to build an exe? Is it hidden in the zip download somewhere? Please make this more clear.

Thanks
Eric

**Comment by fieldfoxWim** [comment]
Would an installer solve your problem?

**Comment by ericfelker** [comment]
An installer would be the most straight forward method in my case at least, but I'm not familiar with what that would take from your side. For the general user, an installer would be the most accessible way to install I would think.

Otherwise, some instruction for taking the zip and getting Super Putty installed from it is what I would need. I looked through all the files and can't figure what I need to do to run the application.

Thanks for the response,

Eric

**Comment by fieldfoxWim** [comment]
Hi Eric

Well maybe you're just downloading the wrong thing.

- go to the release page: https://github.com/jimradford/superputty/releases
- download the latest release: https://github.com/jimradford/superputty/releases/download/1.4.0.6/SuperPuTTY-1.4.0.6.zip
- unzip on your computer
- in the folder you 'll find a file: "SuperPutty.exe"
- double click it

Cheers
Wim

**Comment by ericfelker** [comment]
Yes!

So just to give you an idea of the path I took, which seems reasonable (maybe mskumaran did the same).

Search Google for "SuperPutty Download." First result (https://code.google.com/p/superputty/downloads/list) offers most recent download of May 2013. Decide to look for something newer.

Second result (https://code.google.com/p/superputty/) redirects to Github (https://github.com/jimradford/superputty).

On GitHub, there is a "Download ZIP" button. Sounds like both smkumaran and I grabbed this, then had no idea what to do with it. The correct path would have been to go to releases first, and get a zip from there.

I'm guessing this is obvious to those familiar with GitHub, but not so much for the lay[wo]man. Should this be obvious to all? Did we miss a hint? Can a note be put somewhere to make this clear?

All this aside, thanks for the help and your work on this SSH manager/GUI!!

Eric
